packages/browser/src/Component/THREEUtil.js

```diff
@@ -269,39 +267,46 @@ module.exports = {
         }
         child.geometry.setIndex(indices);
 
-        // Create Outliers
-        const geo = new THREE.EdgesGeometry(child.geometry, threshOldAngle);
+        // Create wireframes
+        const geomEdges = new THREE.EdgesGeometry(
+          child.geometry,
+          threshOldAngle
+        );
         const mat = new THREE.LineBasicMaterial({
           color: 0x000000,
         });
-        const edges = new THREE.LineSegments(geo, mat);
-        edges.userData.isOutlier = true;
-        child.add(edges);
+        const wireframe = new THREE.LineSegments(geomEdges, mat);
+        wireframe.userData.isWireframe = true;
+        child.add(wireframe);
       }
     });
   },
 
   /**
-   * Traverse a THREE.Object3D and append in children for each Object3D with a geometry a THREE.LineSegment representing its wireframe
-   * Each wireframe tile geometry keep the batchid for each wireframe to apply styles on a specific wireframe using a CityObjectID.
-   * This method is slower than appendWireframeToTileset().
-   *
-   * @param {THREE.Object3D} object3D  A 3DObject (should be a tile send by Tilesmanager.EVENT_TILE_LOADED)
+   * Traverse a THREE.Object3D and append in each Object3D children a THREE.LineSegment geometry  representing its wireframe.
+   * Each wireframe geometry will keep the associated attribute value.
+   * This method is slower than appendWireframeToObject3D.
+   * @param {THREE.Object3D} object3D  An Object3D from three
    * @param {number} threshOldAngle  An edge is only rendered if the angle (in degrees) between the face normals of the adjoining faces exceeds this value. default = 1 degree.
+   * @param {string} nameOfGeometryAttribute The attribute used to split each geometry of the BufferGeometry
    */
-  appendWireframeByBatchIDToTileset: function (object3D, threshOldAngle = 30) {
+  appendWireframeByGeometryAttributeToObject3D: function (
+    object3D,
+    threshOldAngle = 30,
+    nameOfGeometryAttribute = '_BATCHID'
```

Comment:
I think this should not have a default parameter value since in three.js batch id means nothing